pynest/nest/lib/hl_api_simulation.py

```diff
@@ -105,7 +110,11 @@ def Run(t):
 
 @check_stack
 def Prepare():
-    """Calibrate the system before a `Run` call. Not needed for `Simulate`.
+    """Calibrate the system before a `Run` call.
+
+    `Prepare` is automatically triggered at beginning of call to `Simulate`:
+
+    Simulate(t): t' = t/m; Prepare(); for _ in range(m): Run(t'); Cleanup()
```

Comment:
```suggestion
    """Calibrate the system before a `Run` call.

    `Prepare` is automatically called by `Simulate` and `RunManager`.
```

Also add `Simulate, RunManager` to See Also below.